docs/source/index.rst

```diff
@@ -387,6 +414,7 @@ Requirements
 Optional
 --------
 
+- `GSTools-Core >= 0.1.0 <https://github.com/GeoStat-Framework/GSTools-Core>`_
```

Comment:
I think it needs at least 0.2.0 because this when the `dim` argument was removed.

Reply:
True!